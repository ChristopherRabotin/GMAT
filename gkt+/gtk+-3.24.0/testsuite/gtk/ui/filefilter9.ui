<!-- test invalid file filter markup element -->
<interface>
  <object class="GtkFileFilter">
    <patterns>
      <pattern><warbl/></pattern>
    </patterns>
  </object>
</interface>
