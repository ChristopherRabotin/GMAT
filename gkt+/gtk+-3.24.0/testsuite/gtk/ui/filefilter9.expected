ERROR: gtk-builder-error-quark 1
.:5:24 Unsupported tag for GtkFileFilter: <warbl>
